File: server/SharingIsCaringAPI/src/SharingIsCaringAPI/Models/Trip.cs
using System;

namespace SharingIsCaringAPI.Models
{
    public class Trip
    {
        public int id { get; set; }
        public string from { get; set; }
        public string to { get; set; }
        public string time { get; set; }
        public string date { get; set; }
        public string passengers { get; set; }
        public int numberOfSeats { get; set; }
        public string car { get; set; }
    }
}

File: client/src/validations/trip.js
import isEmpty from 'lodash/isEmpty';
import Time from 'react-time';

export default function validateInput(data) {
  let errors = {};
  const validator = require('valido');
  if(isEmpty(data.from)) {
    errors.from = 'this field is required';
  }
  if(isEmpty(data.to)) {
    errors.to = 'this field is required';
  }
  if(isEmpty(data.numberOfSeats)) {
    errors.numberOfSeats = 'this field is required';
  }
  if(isEmpty(data.time)) {
    errors.time = 'this field is required';
  }
  if(isEmpty(data.date)) {
    errors.date = 'this field is required';
  } 
  if(isEmpty(data.car)) {
    errors.car = 'this field is required';
  }
  if(!(data.password === data.passwordConfirmation)) {
    errors.passwordConfirmation = 'passwords must match';
  }
  return {
    errors,
    isValid: isEmpty(errors)
  }
}
